src/views/MapView.vue

```diff
@@ -46,11 +46,12 @@ onMounted(() => {
 </script>
 
 <template>
-    <div id="map-view" class="no-print">
-        <LoadingBar v-show="showLoadingBar" />
+    <div id="map-view">
+        <LoadingBar v-show="showLoadingBar" class="no-print" />
```

Comment:
You are breaking the tooltip print here, now with your change when you print the tooltip and you select the text in the tooltip you have part of the view that is printed invisible but selectable.
![image](https://github.com/geoadmin/web-mapviewer/assets/67745584/d8e19bc7-36c1-4112-85fe-dd22a0129f45)

Reply:
See https://github.com/geoadmin/web-mapviewer/pull/905/commits/e1875b1bdc72221295ce53f07d066d0feba08300 from #905